cmd/docs.go

```diff
@@ -67,24 +97,145 @@ func docsMain(cmd *cobra.Command, args []string) exitCode {
 			}
 		}
 		if len(supportedSignals) > 0 {
-			printSignalSet("SUPPORTED SIGNALS", supportedSignals)
+			addSection(docs, stringifySignalSet("SUPPORTED SIGNALS", supportedSignals))
 		}
 		if len(supportedSignalGroups) > 0 {
-			printSignalSet("SUPPORTED SIGNAL GROUPS", supportedSignalGroups)
+			addSection(docs, stringifySignalSet("SUPPORTED SIGNAL GROUPS", supportedSignalGroups))
 		}
 	}
 
+	cmdutil.Println(docs.String())
 	return exitCode{0}
 }
 
-func printSignalSet(setName string, signals []apitypes.SignalSchema) {
-	cmdutil.Println()
-	cmdutil.Printf("%v\n", setName)
+func stringifySupportedAttributes(path string, entry apitypes.Entry) string {
```

Comment:
These could be good functions to unit test.

Reply:
I'll add some tests for them.

Reply:
Done.